webapp/app.py

```diff
@@ -86,23 +87,24 @@ def make_symbol_table(self):
         return self._make_table(
             [descriptor for descriptor in self.symbol_loader.descriptors if descriptor.is_voting and 'NGL' not in descriptor.categories],
             ['address', 'name', 'host', 'balance', 'version', 'voting_end_epoch'],
-            SymbolLoader.version_to_tag)
+            SymbolLoader.version_to_tag,
+            'symbol-table')
 
     def make_nem_table(self):
         return self._make_table(
             [descriptor for descriptor in self.nem_loader.descriptors if descriptor.version],
             ['main_address', 'name', 'host', 'balance', 'version'],
-            NemLoader.version_to_tag)
+            NemLoader.version_to_tag,
+            'nem-table')
 
     @staticmethod
-    def _make_table(descriptors, column_names, version_to_tag):
-        return dash_table.DataTable(
-            data=[
-                {
-                    'tag': version_to_tag(descriptor.version),
-                    **{name: str(getattr(descriptor, name)) for name in column_names}
-                } for descriptor in descriptors
-            ],
+    def _make_table(descriptors, column_names, version_to_tag, id):
+        table = dash_table.DataTable(
```

Comment:
i think you can just return like original version?

Reply:
The identifier was added because we need to be able to track elements to push updates when files are refreshed on disk.